app/uk/gov/hmrc/economiccrimelevyregistration/views/LiabilityDateView.scala.html

```diff
@@ -32,12 +33,12 @@
         @errorSummary(ErrorSummaryViewModel(form))
     }
 
-    <h1 class="govuk-heading-l">@messages("liability.date.h1")</h1>
```

Comment:
Why did you remove this?

Reply:
Because it shouldn't be there. When you set a legend to asPageHeading, it creates the H1 for you.